elements/nuxeo-dropzone/nuxeo-dropzone.html

```diff
@@ -282,7 +309,8 @@
 
       listeners: {
         'batchFinished': 'importBatch',
-        'nx-blob-picked': '_blobPicked'
+        'nx-blob-picked': '_blobPicked',
+        batchFailed: 'importBatch'
```

Comment:
nit: comma in last line
```suggestion
        batchFailed: 'importBatch',
```

Reply:
We are not enforcing comma-dangle on 10.10, so I just kept it consistent with the rest of the styling.